src/utils/translations/id.ts

```diff
@@ -0,0 +1,101 @@
+const id = {
+  // Home Tab
+  Home: "Beranda",
+  "Period in": "Haid akan dimulai dalam",
+  "no info": "tidak ada informasi",
+  Period: "Haid",
+  today: "hari ini",
+  Delay: "Terlambat",
+  Days: "(0)[Hari];(1)[Hari];(2-4)[Hari];(5-20)[Hari];(21)[Hari];(22-24)[Hari];(25-30)[Hari];(31)[Hari];(32-34)[Hari];(35-40)[Hari];",
```

Comment:
You are writing `Hari` in any case, so you just can write `(0-inf)[Hari]` instead

Reply:
okay!